File: app/src/main/java/com/robotnec/budget/core/mvp/view/CategoriesView.java
package com.robotnec.budget.core.mvp.view;

import java.util.List;

import com.robotnec.budget.core.domain.Category;

/**
 * @author zak <zak@swingpulse.com>
 */
public interface CategoriesView extends View {
    void displayCategories(List<Category> categories);
}

File: app/src/main/java/com/robotnec/budget/core/mvp/view/CategoriesView.kt
package com.robotnec.budget.core.mvp.view

import com.robotnec.budget.core.domain.Category

/**
 * @author zak zak@swingpulse.com>
 */
interface CategoriesView : View {
    fun displayCategories(categories: List<Category>)
}
